integration-tests/features/pipelines/create-from-add-options.feature

```diff
@@ -0,0 +1,215 @@
+              And user is able to see workload "<name>" in topology page
+
+        Examples:
+                  | git_url                                 | name          |
+                  | https://github.com/sclorg/nodejs-ex.git | nodejs-ex-git |
+
+
+    #Marking this as manual due to git rate limit  issue
+        @regression @manual
+        Scenario Outline: Add Pipeline option display in git workload for builder image: P-01-TC08
+            Given user is at Import from Git form
+             When user enters Git Repo URL as "<git_url>"
+             Then Add pipeline option is displayed
+
+        Examples:
+                  | git_url                                        |
+                  | https://github.com/sclorg/dancer-ex.git        |
+                  | https://github.com/sclorg/cakephp-ex.git       |
+                  | https://github.com/sclorg/golang-ex.git        |
+                  | https://github.com/sclorg/ruby-ex.git          |
+                  | https://github.com/sclorg/django-ex.git        |
+                  | https://github.com/spring-projects/spring-boot |
+                  | https://github.com/sclorg/nodejs-ex.git        |
+
+
+        @regression @manual
+        Scenario Outline: Add Pipeline option display for dotnet builder image with context directory": P-01-TC09
+            Given user is at Import from Git form
+             When user enters Git Repo URL as "<git_url>"
+              And user selects Show advanced Git options
+              And user clears Context dir field
+              And user enters Context dir as "<dir_name>"
+             Then Add pipeline option is displayed
+
+        Examples:
+                  | git_url                                                   | dir_name |
+                  | https://github.com/redhat-developer/s2i-dotnetcore-ex.git | /app     |
+
+
+        @regression @manual
+        Scenario Outline: Add Pipeline option doesn't display for server related git urls: P-01-TC10
+            Given user is at Import from Git form
+             When user enters Git Repo URL as "<git_url>"
+              And user clicks on "Edit import strategy"
+              And user selects Import Strategy as Builder Image
+              And user selects "<builder_image>" builder image
+             Then git url "<git_url>" gets Validated
+              And user is able to see info message "<message>" in Pipelines section
+
+        Examples:
+                  | git_url                                | builder_image | message                                                                         |
+                  | https://github.com/sclorg/httpd-ex.git | httpd         | There are no pipeline templates available for Httpd and Deployment combination. |
+                  | https://github.com/sclorg/nginx-ex.git | Nginx         | There are no pipeline templates available for Nginx and Deployment combination. |
+
+
+        @regression @odc-6372
```

Comment:
```
$ yarn gherkin-lint
yarn run v1.22.22
$ ./node_modules/.bin/gherkin-lint -c ./integration-tests/.gherkin-lintrc ./integration-tests/features
/home/ppitonak/workspace/console-plugin/integration-tests/features/pipelines/create-from-add-options.feature
  158    Not allowed tag @odc-6372 on Scenario Outline    allowed-tags
  176    Not allowed tag @odc-6372 on Scenario Outline    allowed-tags
  195    Not allowed tag @odc-7128 on Scenario Outline    allowed-tags
```

Reply:
What does `odc-6372` stand for? Jira story?

Reply:
Yes, it's for the respective JIRA story. I can remove these.